piperider_cli/cloud_connector.py

```diff
@@ -281,8 +318,19 @@ def compare_reports(base=None, target=None, tables_from='all', summary_file=None
             console.rule('Please login PipeRider Cloud first', style='red')
             return 1
 
-        base_id = get_run_report_id(base)
-        target_id = get_run_report_id(target)
+        if base is None or target is None:
+            base, target = select_cloud_reports(base=base, target=target)
```

Comment:
Just a discussion:
what do you think if `select_cloud_reports` returns cloud report id directly?

Reply:
In this function, I think return report id should be more reasonable. I will also rename the function `select_cloud_reports` to `select_cloud_report_ids`

Reply:
Fix